crates/matrix-sdk-base/src/rooms/mod.rs

```diff
@@ -288,6 +295,36 @@ impl BaseRoomInfo {
             self.rtc_member.retain(|_, member_event| member_event.event_id() != Some(redacts));
         }
     }
+
+    pub fn handle_notable_tags(&mut self, tags: &Tags) {
+        let mut notable_tags = RoomNotableTags::empty();
+
+        if tags.contains_key(&TagName::Favorite) {
```

Comment:
haha, now Ruma needs an update for `TagName::Favourite` too ^^

Reply:
Indeed!